ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/SchoolPerformance/SchoolPerformanceIntegrationTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using ApplyToBecome.Data.Models;
+using ApplyToBecomeInternal.Extensions;
+using ApplyToBecomeInternal.Tests.Helpers;
+using AutoFixture;
+using AutoFixture.Kernel;
+using FluentAssertions;
+using System;
+using System.Net.Http;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using WireMock.Server;
+using Xunit;
+using SchoolPerformanceModel = ApplyToBecome.Data.Models.SchoolPerformance;
+
+namespace ApplyToBecomeInternal.Tests.Pages.SchoolPerformance
+{
+	public class SchoolPerformanceIntegrationTests : IClassFixture<IntegrationTestingWebApplicationFactory>
+	{
+		private readonly HttpClient _client;
+		private readonly WireMockServer _server;
+
+		public SchoolPerformanceIntegrationTests(IntegrationTestingWebApplicationFactory factory)
+		{
+			_client = factory.CreateClient();
+			_server = factory.WMServer;
+		}
+
+		[Fact]
+		public async Task Should_display_school_performance_ofsted_information()
```

Comment:
might be good to have a test around navigating to and from the school performance page from the task list?